nexus-config/src/nexus_config.rs

```diff
@@ -174,6 +174,9 @@ pub struct DeploymentConfig {
     /// Dropshot configuration for internal API server.
     #[schemars(skip)] // TODO we're protected against dropshot changes
     pub dropshot_internal: ConfigDropshot,
+    /// Dropshot configuration for lockstep API server.
+    #[schemars(skip)] // TODO we're protected against dropshot changes
+    pub dropshot_lockstep: ConfigDropshot,
```

Comment:
I'm afraid this might be a flag day between Nexus and Sled Agent (because it writes out this config file).  But I guess it's okay because Sled Agent is always updated before Nexus, so it will just start supplying this.

Reply:
Yeah, it would be. I think if we're not planning on running Nexus-driven update on dogfood before this lands, we're fine, but otherwise I could make this an `Option` and bubble that all the way through? Would be obnoxious but theoretically fine.